### PR TITLE
fix translation

### PR DESCRIPTION
**Pull request by reindex-ot**
Japanese Translation Adjustments and Fixes

### CODE REVIEW

**Comment by tiann** [comment]
Please resolve the conflicts

**Comment by reindex-ot** [comment]
resolved.

File: manager/app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   <string name="home_working_version">バージョン: %d</string>
   <string name="home_unsupported">非対応</string>
-  <string name="home_unsupported_reason">KernelSUは現在、GKI Kernelのみサポートをしています</string>
+  <string name="home_unsupported_reason">KernelSUは現在、GKIカーネルのみサポートをしています</string>
   <string name="home_copied_to_clipboard">クリップボードにコピーしました</string>
   <string name="home_support">対応</string>
   <string name="home_kernel">カーネル</string>
   <string name="home_arch">アーキテクチャ</string>
-  <string name="home_manager_version">バージョン</string>
+  <string name="home_version">バージョン</string>
```
Comment (tiann) [inline manager/app/src/main/res/values-ja/strings.xml:15]: This should be `home_manager_version`

**Review by tiann — CHANGES_REQUESTED** [review]

File: manager/app/src/main/res/values-ja/strings.xml (hunk 1 above)
Comment (reindex-ot) [reply manager/app/src/main/res/values-ja/strings.xml]: sorry...fixed.

**Comment by tiann** [comment]
Thanks for your contribution!

**Review by tiann — APPROVED** [review]

**Pull request merged**